torch/distributed/_composable/fsdp/_fsdp_common.py

```diff
@@ -56,6 +57,13 @@ def __post_init__(self):
         super(DDPMeshInfo, self).__post_init__()
 
 
+class TrainingState(Enum):
+    FORWARD = auto()
+    PRE_BACKWARD = auto()
+    POST_BACKWARD = auto()
+    IDLE = auto()
```

Comment:
can we add some comment about the IDLE state and when we will be in the idle state? i.e. it would be idle after the FSDP initialization but before runtime?